test/libyul/yulSyntaxTests/eof/eof_builtins_names_reserved.yul

```diff
@@ -0,0 +1,20 @@
+object "a" {
+    code {
+        function dataloadn() {}
+        function rjump() {}
+        function rjumpi() {}
+        function callf() {}
+        function jumpf() {}
+        function retf() {}
```

Comment:
Are any of these implemented in EOF, i.e. callable, like ext* calls?

Reply:
They are not callable from yul level, but legacy `jump` and `jumpi` are reserved but also not callable from yul. So I assumed they should be reserved also.